internal/pkg/store/project.go

```diff
@@ -25,9 +25,13 @@ func (s *Store) CreateProject(project *archer.Project) error {
 	}
 
 	if project.Domain != "" {
-		in := &route53domains.GetDomainDetailInput{DomainName: aws.String(project.Domain)}
-		if _, err := s.domainsClient.GetDomainDetail(in); err != nil {
-			return fmt.Errorf("get domain details for %s: %w", project.Domain, err)
+		in := &route53.ListHostedZonesByNameInput{DNSName: aws.String(project.Domain)}
+		resp, err := s.hostedZoneClient.ListHostedZonesByName(in)
```

Comment:
This is probably fine, but we're not paginating in case there are more responses.

Reply:
Yeah you are right. I should add pagination.